plugins/woocommerce/client/admin/client/settings-payments/components/country-selector/country-selector.tsx

```
@@ -253,12 +256,98 @@ export const CountrySelector = < ItemType extends Item >( {
+					} );
+					break;
+				case 'Enter':
+					event.preventDefault();
+					// Use highlighted item if available, otherwise use current selection.
+					const itemToApply =
+						keyboardHighlightIndex !== null &&
+						keyboardHighlightIndex >= 0 &&
+						keyboardHighlightIndex < itemCount
+							? itemsArray[ keyboardHighlightIndex ]
+							: selectedItem;
+					if ( itemToApply ) {
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)